File: 240309/999 또는 -999/999-or-999.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int arr[100];
    for(int i=0;i<100;i++){
        cin>>arr[i];
        if(arr[i]==999||arr[i]==-999) break;
    }
    int max_val=arr[0];
    int min_val=arr[0];

    for(int i=1;i<100;i++){
        if(arr[i]==999||arr[i]==-999) break;

        if(arr[i]>max_val) max_val=arr[i];
        if(min_val>arr[i]) min_val=arr[i];
    }
cout<<max_val<<" "<<min_val;
return 0;
}

File: 240606/999 또는 -999/999-or-999.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    Scanner sc=new Scanner(System.in);

    int[]arr=new int[100];
    int maxval=arr[0];
    int minval=arr[0];

    for(int i=0;i<100;i++){
        arr[i]=sc.nextInt();
        if(arr[i]==999||arr[i]==-999)
        break;
        if(arr[i]>maxval){
            maxval=arr[i];
        }
        if(minval>arr[i]){
        minval=arr[i];}
    }
    System.out.print(maxval+" "+minval);
    }
}